src/inmanta/compiler/help/templates/module_v2_in_v1_path.j2

```diff
@@ -0,0 +1,11 @@
+This error occurs when a v2 module was found in v1 modules path. To resolve this you should either convert this module to be v1
+or install it as a v2 module and set up your project accordingly.
+
+If you want to use the module as a v1 module, make sure to use the v1 cookiecutter template to create new modules.
+
+If you want to use the module as a v2 module:
+{% if project and not v2_source_configured -%}
+- set up your project with a module source of type "package" (see documentation)
+{% endif -%}
+- if you would like to work in editable mode on a local copy of the module, run `inmanta module install {{ path }}`
+- run `inmanta module add {{ name }}` to add the module as a dependency and to install the module if required.
```

Comment:
I don't think it is. Even if you're working in editable mode, you'd want to add the dependency to the project, wouldn't you?

Reply:
The problem with switching the order is that if the package does not exist yet on an index, the `inmanta module add` would fail. By having this order, you first install everything you want locally, then run the command to add it to the project. Same goes for `inmanta project install`, it would just install any missing dependencies, leaving alone what has already been installed.

Reply:
Yes. I agree that the command can stay. I would also make sure that the `inmanta module add` command logs to the console when it overrides an existing installation.

Reply:
Good suggestion, I'll add that.

Reply:
Though I guess it might not be straightforward, since it is pip that does this, not us.